src/main/java/mate/academy/review/analysis/entity/Product.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.review.analysis.entity;
+
+import java.util.Set;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "products")
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String productId;
```

Comment:
@ggazila what is the difference between id and productId? Is the productId unique?